regress/pesterTests/SSH.Tests.ps1

```diff
@@ -1,78 +1,56 @@
 ﻿If ($PSVersiontable.PSVersion.Major -le 2) {$PSScriptRoot = Split-Path -Parent $MyInvocation.MyCommand.Path}
-Import-Module $PSScriptRoot\CommonUtils.psm1 -Force
+. $PSScriptRoot\TestUtils.ps1
```

Comment:
This requires the test to be run from its source folder. I had to do this in my common.ps1 to overcome this limitation
. (Join-Path (Split-Path $MyInvocation.MyCommand.Path) common.ps1)